setup.py

```diff
@@ -19,6 +19,7 @@
     install_requires=[
         'pylint-plugin-utils>=0.5',
         'pylint>=2.0',
+        'Faker==5.0.1',
```

Comment:
Thanks for digging into this. Can you also remove the test jobs for Python 3.5 because Faker includes:
`python_requires=">=3.6"` and that causes these jobs to fail anyway.

Reply:
Another question: Does v 5.0.2 or any of the others work or this is the last one which works fine ?

Reply:
Just removed python 3.5 from the tests.  Faker 5.0.1 appears to be the last version that works. [This PR](https://github.com/joke2k/faker/pull/1339/files) which landed in v5.0.2 is the one that is causing issues.